packages/opencensus-core/test/test-sampler.ts

```diff
@@ -90,6 +35,20 @@ describe('Sampler', () => {
       const samplerShouldSampler = sampler.shouldSample(root.traceId);
       assert.ok(samplerShouldSampler);
     });
+    it('should return a always sampler', () => {
+      const root = new RootSpan(tracer);
+      const sampler = new Sampler().probability(1);
+      const samplerShouldSampler = sampler.shouldSample(root.traceId);
+      assert.strictEqual(sampler.description, 'always');
+      assert.ok(samplerShouldSampler);
+    });
+    it('should return a always sampler', () => {
```

Comment:
Use different test case messages for these two tests (same for `never sampler`)

Reply:
Done!